librz/core/cmd_analysis.c

```diff
@@ -9585,7 +9591,7 @@ RZ_IPI RzCmdStatus rz_analysis_function_vars_type_handler(RzCore *core, int argc
 	return RZ_CMD_STATUS_OK;
 }
 
-RZ_IPI RzCmdStatus rz_analysis_function_vars_xrefs_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
+RZ_IPI RzCmdStatus RZ_ANALYSIS_FUNCTUIN_VARS_XREFS_HANDLER(RzCore *core, int argc, const char **argv, RzOutputMode mode, bool use_afvxa, bool use_afvxv) {
```

Comment:
1. A typo
2. Why CAPS?

Reply:
Sorry, I don't know what CAPS means. My idea is to rename the original `rz_analysis_function_vars_xrefs_handler` to `RZ_ANALYSIS_FUNCTION_VARS_XREFS_HANDLER` and add three functions for `afvx`, `afvxa` and `afvxv` to call `RZ_ANALYSIS_` `FUNCTION_VARS_XREFS_HANDLER`

Reply:
CAPS means `all uppercase`. why you changed the name of the function to all uppercase letters? normally uppercased names are assigned to macros with constants

Reply:
Thanks, I will rename the function in another way.